_includes/cli-background-note.md

```diff
@@ -0,0 +1 @@
+{{site.data.alerts.callout_info}} `--background` is not full daemonization (it doesn't write PID files, among other things).  Tools like [daemon(8)](https://www.freebsd.org/cgi/man.cgi?query=daemon&sektion=8) are recommended for deployments that require daemonization. {{site.data.alerts.end}}
```

Comment:
In Ben's original answer, he ends with:
> (but non-daemon-based process managers are even better).

Any reason we're not mentioning that?

Also, what should we do about the user of `--background` in all of the on-premises and cloud deployment tutorials?

Reply:
We can make this more precise and accurate. In particular, the existence of the flag `--pid-file` (even since 1.0) invalidates (part of) the argument. Try this:

"""
 `--background` is suitable to write automated tests suites or maintenance procedures that need a temporary server process running in the background. It is not intended for use to start a long-running server, because it does not fully detach from the controlling terminal. Consider using a service manager or a tool like [daemon(8)](https://www.freebsd.org/cgi/man.cgi?query=daemon&sektion=8) instead.
"""